File: backend/src/main/java/pl/wwf/nowaste/domain/product/ratings/Rating.java
package pl.wwf.nowaste.domain.product.ratings;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Embeddable;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Embeddable
public class Rating {

    private Integer boxReusable;

    private Integer boxRecycable;

    private Integer boxFromRecycling;

    private Integer productReusable;

    private Integer productRecycable;

    private Integer productFromRecycling;

    private Integer repairable;

}

File: mobile/android/app/src/main/java/com/hackyeah/nowaste/data/model/Rating.kt
package com.hackyeah.nowaste.data.model

import java.io.Serializable
import java.math.BigDecimal

class Rating(
    val boxReusable: BigDecimal?,
    val boxRecycable: BigDecimal?,
    val boxFromRecycling : BigDecimal?,
    val productReusable: BigDecimal?,
    val productRecycable: BigDecimal?,
    val productFromRecycling: BigDecimal?,
    val repairable: BigDecimal?
) : Serializable